cryptofeed/exchanges/binance.py

```diff
@@ -261,26 +272,107 @@ async def _snapshot(self, pair: str) -> None:
         url = f'{self.rest_endpoint}/depth?symbol={pair}&limit={max_depth}'
         resp = await self.http_conn.read(url)
         resp = json.loads(resp, parse_float=Decimal)
+        return resp
+
+    async def _reset_snapshot(self, pair: str) -> None:
+        resp = await self._fetch_snapshot(pair)
 
         std_pair = self.exchange_symbol_to_std_symbol(pair)
         self.last_update_id[std_pair] = resp['lastUpdateId']
-        self._l2_book[std_pair] = OrderBook(self.id, std_pair, max_depth=self.max_depth, bids={Decimal(u[0]): Decimal(u[1]) for u in resp['bids']}, asks={Decimal(u[0]): Decimal(u[1]) for u in resp['asks']})
+        self._l2_book[std_pair] = OrderBook(self.id, std_pair, bids={Decimal(u[0]): Decimal(u[1]) for u in resp['bids']}, asks={Decimal(u[0]): Decimal(u[1]) for u in resp['asks']})
```

Comment:
I don't think we want to set `max_depth` on the order book. That will always truncate the book for that size, which gets rid of useful updates higher up in the book and can contribute to the order book getting out of date.

Reply:
max depth doesnt truncate, unless you set truncate to True, which nothing does. The data is still there and can move up if necessary